symengine/lib/symengine_wrapper.pyx

```diff
@@ -37,6 +37,12 @@ cdef c2py(RCP[const symengine.Basic] o):
     elif (symengine.is_a_Pow(deref(o))):
         r = Basic.__new__(Pow)
     elif (symengine.is_a_Integer(deref(o))):
+        if (deref(symengine.rcp_static_cast_Integer(o)).is_zero()):
+            return S.Zero
+        elif (deref(symengine.rcp_static_cast_Integer(o)).is_one()):
+            return S.One
+        elif (deref(symengine.rcp_static_cast_Integer(o)).is_minus_one()):
+            return S.NegativeOne
```

Comment:
Like here we are directly returning `Singleton` attributes. Where am I supposed to create the `Integer(0)` that `S.Zero` will return?